src/windows/MediaProxy.js

```diff
@@ -33,8 +42,11 @@ module.exports = {
     create:function(win, lose, args) {
         var id = args[0];
         var src = args[1];
+        var createAudioNode = args[2];
         var thisM = Media.get(id);
 
+        src = setTemporaryFsByDefault(src);
```

Comment:
Would prefer this line immediately follow "var src = args[1]" because it is easy to miss this mutation.